src/plugins/intel_gpu/src/plugin/ops/constant.cpp

```diff
@@ -88,38 +88,26 @@ static void create_data(ProgramBuilder& p, const ov::Shape& const_shape, const s
     cldnn::primitive_id constPrimID;
     auto data = op->get_data_ptr<char>();
 
-    const auto cache_key = std::make_tuple(data, const_shape, op->get_output_element_type(0));
```

Comment:
I think we should keep this logic because the new pass only handles constants with limited size, so some buffers still can be optimized using this approach